src/2.0/schema/schema.json

```diff
@@ -221,6 +221,21 @@
       },
       "additionalProperties": true
     },
+    "chainDetail": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Which blockchain being used"
+        },
+        "chainId": {
+          "type": "string",
```

Comment:
number?

Reply:
We could keep it as string too since chain ID is 256 bit though so far haven't seen any networks using super huge IDs.😂